File: Esquenta/Entities/PeriodoVenda.cs
using Esquenta.Entities.Interfaces;
using System;

namespace Esquenta.Entities
{
    public class PeriodoVenda : IBaseEntity
    {
        public virtual int Id { get; protected set; }
        public virtual DateTime DataInicial { get; set; }
        public virtual DateTime? DataFinal { get; set; }
        public virtual decimal ValorCaixa { get; set; }
        public virtual decimal ValorEmAberto { get; set; }
    }
}

File: V2-Node/esquenta.v2/server/models/periodovenda.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  var periodoVenda = sequelize.define('periodoVenda', {
    dataInicial: DataTypes.DATE,
    dataFinal: DataTypes.DATE,
    valorEmAberto: DataTypes.DECIMAL
  }, {});
  periodoVenda.associate = function(models) {
    // associations can be defined here
  };
  return periodoVenda;
};